test/ziggurat/middleware/batch/batch_proto_deserializer_test.clj

```diff
@@ -0,0 +1,53 @@
+(ns ziggurat.middleware.batch.batch-proto-deserializer-test
+  (:require [clojure.test :refer :all])
+  (:require [ziggurat.middleware.batch.batch-proto-deserializer :refer :all]
+            [protobuf.core :as proto])
+  (:import (flatland.protobuf.test Example$Photo Example$Photo$Tag)))
+
+(deftest batch-proto-deserializer-test
```

Comment:
Can we combine two deftest into one deftest with two testing? Because there is a lot of duplication in setting them up.

Reply:
sure.